test/non_ssl.jl

```diff
@@ -228,9 +228,14 @@ end
 
     # it is not the same file when downloading in ascii mode
     options = RequestOptions(; opts..., ssl=false, active_mode=false)
-    resp = ftp_get(options, byte_file; mode=ascii_mode)
-    bytes = read(resp.body)
-    @test bytes == filter(!cr, download_bytes)
+
+    # Our implementation currently checks size which does not work in ASCII mode on FTPServer
+    # https://github.com/invenia/FTPClient.jl/issues/113
+    @test_skip begin
```

Comment:
should these maybe be test_broken instead of test_skip?

Reply:
`@test_broken` will error if the case succeeds.

```julia
using Test

f(a,b) = a+b

julia> @test_broken f(1,1) == 3
Test Broken
  Expression: f(1, 1) == 3

julia> @test_broken f(1,1) == 2
Error During Test at REPL[6]:1
 Unexpected Pass
 Expression: f(1, 1) == 2
 Got correct result, please change to @test if no longer broken.

ERROR: There was an error during testing
```

Reply:
yeah the idea for test broken is that it would fail on a succeed if something in the future fixed the issue, but you're right I guess it doesn't work in this situation.

If we can't use test broken, then how do we get automatically notified when it starts working again to unskip those tests? Is the libcurl issue potentially going to just start working again in the future? Could we add a `test_broken` that specifically tests that we get an error here so that if it  starts working again we get notified?

Reply:
Ah I'd misremembered! `@test_broken` does indeed work in this case